torcheeg/models/cnn/fbcnet.py

```diff
@@ -207,4 +178,144 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         return x
 
     def feature_dim(self):
-        return self.num_S * self.in_channels * self.stride_factor
\ No newline at end of file
+        return self.num_S * self.in_channels * self.stride_factor
+
+
+class FBMSNet(nn.Module):
+    r'''
+    FBMSNet, a novel multiscale temporal convolutional neural network for MI decoding tasks, employs Mixed Conv to extract multiscale temporal features which  enhance the intra-class compactness and improve the inter-class separability with the joint supervision of the center loss andcenter loss.
+
+    - Paper: FBMSNet: A Filter-Bank Multi-Scale Convolutional Neural Network for EEG-Based Motor Imagery Decoding
+    - URL: https://ieeexplore.ieee.org/document/9837422
+    - Related Project: https://github.com/Want2Vanish/FBMSNet
+
+    Below is a recommended suite for use in emotion recognition tasks:
+
+    .. code-block:: python
+
+        band_dict = { str(i):[4*i,4*(i+1)] for i in range(1,10) }   
```

Comment:
"Provide a manual definition to allow for intuitive understanding by the user, for example, `{'theta': [4, 8], ...}`.